GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeViewController.swift

```diff
@@ -0,0 +1,103 @@
+import UIKit
+
+final class VisitorCodeViewController: UIViewController {
+
+    var timer: Timer?
```

Comment:
Are these properties needed to be internal?

Reply:
fixed [fc50a16](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/fc50a168934e072d2a4b217c22bcbdc235a31d95)